.github/workflows/storybook_deploy_main.yml

```diff
@@ -0,0 +1,28 @@
+name: Deploy Main Storybook
+
+on:
+  schedule:
+    - cron: '0 7 * * *'
+
+jobs:
+  deploy_main_storybook:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        clean: false
+    - name: Run npm install
+      run: npm install
+    - name: Build Storybook
+      run: npm run build-storybook -- --output-dir=storybooks/main
+    - name: Deploy Storybook to surge.sh
+      id: surge_deploy
+      env:
+        SURGE_LOGIN: ${{ secrets.SURGE_LOGIN }}
+        SURGE_TOKEN: ${{ secrets.SURGE_TOKEN }}
+      if: ${{ env.SURGE_TOKEN }}
+      run: |
+        npm install --save-dev surge
+        node_modules/.bin/surge --project storybooks/main --domain foreman.surge.sh
```

Comment:
Can we use `npx surge` instead of `node_modules/.bin/surge`? [More info is here](https://blog.npmjs.org/post/162869356040/introducing-npx-an-npm-package-runner) if you are not familiar, its a binary that comes with npm and allows one to use local package binaries.